Colibri.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EjercicioDos
+{
+    internal class Colibri : Ave, Muerte
+    {
+        //se sobre escribe metodo comer
+        public override void comer()
+        {
+            Console.WriteLine("el colibri come");
+        }
+        //se crea comportmiento diferente
+        public void volar()
+        {
+            Console.WriteLine("el colibri vuela");
+        }
+
+        //se reciben los parametros de muerte y se calcula cuanto vivio
+        public string Morir(int nacimiento, int faleccimiento, string causa)
```

Comment:
Type error on faleccimiento please check it

Reply:
has been corrected